viewing_party/party.py

```diff
@@ -0,0 +1,128 @@
+def create_movie(title, genre, rating):
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    if new_movie["title"] == None:
+        return None
+    elif new_movie["genre"] == None:
+        return None
+    elif new_movie["rating"] == None:
+        return None
+    return new_movie
```

Comment:
👍 This works, but can be done a bit more compactly

```suggestion
def create_movie(title, genre, rating):
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating,
        }
```